### PR TITLE
Add support for setAttribute via setConnectionParameter in ADOdb #247

### PR DESCRIPTION
**Pull request by mnewnham**
The PDO driver now supports the setAttribute option via ADOdb setConnectionParameter method

### CODE REVIEW

File: drivers/adodb-pdo_sqlsrv.inc.php [hunk 1]
```diff
@@ -45,5 +48,146 @@ function ServerInfo()
...
+		* which crashes the method
+		*/
+		if ($fieldOffset == -1)
+			$fieldOffset++;
+
+		if (isset($fieldObjects[$fieldOffset]))
+			/*
+		    * Look for cached field offset
+			*/
+			return $fieldObject[$fieldOffset];
```
Comment (dregad) [inline drivers/adodb-pdo_sqlsrv.inc.php]: Shouldn't that be `$fieldObjects` ?

File: drivers/adodb-pdo_sqlsrv.inc.php [hunk 2]
```diff
@@ -45,5 +48,146 @@ function ServerInfo()
...
```
Comment (dregad) [inline drivers/adodb-pdo_sqlsrv.inc.php]: Same comment as [above](https://github.com/ADOdb/ADOdb/pull/249/files#r64607035)

Comment (dregad) [inline drivers/adodb-pdo_sqlsrv.inc.php]: Same comment as [above](https://github.com/ADOdb/ADOdb/pull/249/files#r64607035)

**Comment by dregad** [comment]
@mnewnham your feedback on the above would be appreciated, so I can merge this.

**Comment by mnewnham** [comment]
Yes, can you tweak this for me?

**Comment by dregad** [comment]
No problem will do

**Comment by dregad** [comment]
This was fixed in 76617d1312eaca62e258353bda8b83a002a65eb2

**Pull request closed without merging**